drivers/watchdog/iwdg_stm32.c

```diff
@@ -101,60 +108,39 @@ static int iwdg_stm32_set_config(struct device *dev,
 	/* Neither STM32F1X nor STM32F4 series supports window option. */
 	LL_IWDG_SetWindow(iwdg, 0x0FFF);
```

Comment:
It is a little bit confusing. Currently there is no option added in the Kconfig to activate Window Mode. So setting it to the default value 0x0000 0FFF in fact does nothing. It neither modifies Window register IWDG_WINR nor reloads the Reload Register IWDG_RLR.
Maybe it would be worth to add appropriate config option to Kconfig.stm32?

Reply:
Would it be good to just remove this for now, and someone can add the feature in the future?